docs/src/components/Layout/GlobalNav/GlobalNav.module.scss

```diff
@@ -41,143 +44,187 @@
 
 .dev-center-logo {
   color: var(--amplify-colors-neutral-10);
-  font-size: 1rem;
+  font-size: var(--amplify-font-sizes-large);
   white-space: nowrap;
 }
 
-.nav-container {
-  display: flex;
-  height: var(--dev-center-nav-height);
-  align-items: center;
-  justify-content: space-between;
-}
-
-.mobile-nav-container {
-  display: flex;
-  height: var(--dev-center-nav-height);
-  align-items: stretch;
-  justify-content: space-between;
-}
-
 .nav-links-container {
-  height: var(--dev-center-nav-height);
+  z-index: 200;
+  width: 100%;
+  position: absolute;
+  top: 0;
   align-items: center;
   justify-content: space-between;
   align-items: stretch;
-  padding-block: 0;
-  padding-inline: var(--amplify-space-large);
-}
-
-.mobile-nav-menu-container {
-  z-index: 1000;
-  background-color: var(--amplify-colors-neutral-90);
-  position: absolute;
-  display: flex;
+  height: auto;
   flex-direction: column;
-  width: 100%;
+  padding: 0;
+  background-color: var(--amplify-colors-neutral-90);
+  gap: initial;
+  & .left-nav-links,
+  .right-nav-links {
+    flex-direction: column;
+    gap: initial;
+  }
 }
 
-.mobile-nav-icons-container {
-  margin-left: auto;
-  margin-right: auto;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-around;
-
-  svg {
-    display: block;
-    margin: auto;
+.nav-expander {
+  &:hover,
+  :active {
+    border: none;
+    box-shadow: none;
   }
-}
 
-.mobile-nav-menu-items {
-  border-top: 1px solid var(--navbar-border-color);
-  padding-inline: var(--amplify-space-small);
-  height: var(--dev-center-nav-height);
-  width: 100%;
+  &:focus {
+    box-shadow: 0 0 0 2px var(--amplify-colors-neutral-10);
+  }
 }
 
 .nav-menu-item {
-  padding: 0 var(--amplify-space-xs);
   display: flex;
   align-items: center;
   height: 100%;
-  font-size: var(--amplify-font-sizes-small);
+  width: 100%;
+  font-size: var(--amplify-font-sizes-medium);
   text-decoration: none;
   color: var(--amplify-colors-neutral-40) !important;
```

Comment:
I see we have a lot of `!important`, even though they are not added in this PR, since they are a part of the same project. Shall we refactor them?

Reply:
Thanks for looking @0618! I think these `!important`'s were kept because without them the some of the colors do not take effect because of the UI Docs theme. We're using this class for the `a` tags and so the pseudo selectors we're using get overwritten with the the amplify UI theme for `.amplify-link`. 

This component should be in its own package but since we are including it as a component at the moment, these `!important` help keep the dev center colors from not being overwritten. Is that okay with you?

Reply:
Can you add more classes to it instead? Like `.amplify-link.GlobalNav_nav-menu-item {...}`?